app/domain/util/stubs/deep_double.rb

```diff
@@ -0,0 +1,72 @@
+module Util
+  module Stubs
+    class DeepDouble
```

Comment:
This could benefit from some documentation comments describing what it is, why it exists, and how to use it.